### PR TITLE
Write data from InfluxDB OSS to InfluxDB Cloud

### LINKED ISSUE #3322: Selective writing of Data to Cloud (OSS content)
Selective writing of Data to Cloud (OSS content)
a. Using Flux to --- make sure to provide caveats similar to those contained within this blog https://www.influxdata.com/blog/how-to-consolidate-oss-data-into-cloud-account/ (the blog outlines a very selective construction of line protocol and leverages HTTP POST). Flux with the to enhancements will allow for more data to be transmitted with less code. However, the limitations around rate limits, volume, retries, etc. will all still apply.

cc @timhallinflux for review

### PR DESCRIPTION
**Pull request by sanderson**
Closes #3322

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.1/write-data/oss-to-cloud.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+Use [`map()`](/flux/v0.x/stdlib/universe/map/), [`set()`](/flux/v0.x/stdlib/universe/set/),
+[`aggregateWindow()`](/flux/v0.x/stdlib/universe/aggregatewindow/)
+[aggregate](/flux/v0.x/function-types/#aggregates) or
+[selector functions](/flux/v0.x/function-types/#selectors), or other Flux
+functions to process and shape data as needed.
+
+{{% note %}}
+The shape of the processed data determines which `to()` function to use.
+_See [Identify which to() function to use](#identify-which-to-function-to-use)_. 
+{{% /note %}}
```
Comment (kelseiv) [inline content/influxdb/v2.1/write-data/oss-to-cloud.md:187]: feels a bit circular/overlapping 

Ideally, we'd link to a separate procedure for info about "processing" data that covers all aforementioned functions. 

Perhaps eventually here :
(/influxdb/cloud/process-data/get-started/#process-or-transform-your-data)...once we land examples for map/set/aggregrate/selectors?

File: content/influxdb/v2.1/write-data/oss-to-cloud.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+        bucket: "example-cloud-bucket",
+        host: "https://cloud2.influxdata.com",
+        org: "example-org",
+        token: cloudToken,
+    )
+```
+
+## Automate writing data from InfluxDB OSS to InfluxDB Cloud
+To automatically and routinely write data from InfluxDB OSS to InfluxDB Cloud,
+[Create a task](/influxdb/v2.1/process-data/manage-tasks/create-task/) in your
```
Comment (kelseiv) [inline content/influxdb/v2.1/write-data/oss-to-cloud.md:237]:
```suggestion
[create a task](/influxdb/v2.1/process-data/manage-tasks/create-task/) in your
```

File: content/influxdb/v2.1/write-data/oss-to-cloud.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+to [routinely write data to InfluxDB Cloud](#automate-writing-data-from-influxdb-oss-to-influxdb-cloud).
+
+{{% cloud %}}
+#### InfluxDB Cloud rate limits
+Write requests to InfluxDB Cloud are subject to the rate limits associated with your
+[InfluxDB Cloud pricing plan](/influxdb/cloud/account-management/pricing-plans/).
+{{% /cloud %}}
+
+1.  Query data from InfluxDB OSS.
+2.  _(Optional)_ Filter or process data to write to InfluxDB Cloud.
```
Comment (kelseiv) [inline content/influxdb/v2.1/write-data/oss-to-cloud.md:27]: maybe link to filter function would be helpful...and/or link process data procedure?

File: content/influxdb/v2.1/write-data/oss-to-cloud.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+3.  Use `to` or `experimental.to` to write data to InfluxDB Cloud.
+    For most use cases, `to()` is the correct function to use, but depending on
+    the structure of the data you're writing, `experimental.to` may be required.
+    
+    **Use the following guidelines**:
+    
+    - **to()**: Write data that is structured with the field key in the `_field`
+      column and the field value in the `_value` column.
+    - **experimental.to()**: Write data with column names that should be used as
+      field keys and column values that should be used as field values.
```
Comment (kelseiv) [inline content/influxdb/v2.1/write-data/oss-to-cloud.md:37]:
```suggestion
- **to()**: Use to write data in field keys to the `_field` column and field values to the `_value` column.

- **experimental.to()**: Use to write data in column names to corresponding field keys and column values to field values.
```

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.1/write-data/oss-to-cloud.md (hunk 3 above)
Comment (sanderson) [inline content/influxdb/v2.1/write-data/oss-to-cloud.md:27]:
```suggestion
2.  _(Optional)_ [Filter](/{{% latest "flux" %}}/stdlib/universe/filter/) or process data to write to InfluxDB Cloud.
```

File: content/influxdb/v2.1/write-data/oss-to-cloud.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+2.  _(Optional)_ [Filter](/{{% latest "flux" %}}/stdlib/universe/filter/) or process data to write to InfluxDB Cloud.
+3.  Use `to` or `experimental.to` to write data to InfluxDB Cloud.
+    For most use cases, `to()` is the correct function to use, but depending on
+    the structure of the data you're writing, `experimental.to` may be required.
+    
+    **Use the following guidelines**:
+    
+- **to()**: Use to write data in field keys to the `_field` column and field values to the `_value` column.
+
+- **experimental.to()**: Use to write data in column names to corresponding field keys and column values to field values.
```
Comment (sanderson) [inline content/influxdb/v2.1/write-data/oss-to-cloud.md:36]:
```suggestion
    - **to()**: Use to write data in field keys to the `_field` column and field values to the `_value` column.

    - **experimental.to()**: Use to write data in column names to corresponding field keys and column values to field values.
```

**Pull request merged**